src/core/utilities.scss

```diff
@@ -2,78 +2,149 @@
 
 .container {
   max-width: $breakpoint-medium;
-  margin: 0 auto;
+  margin-left: auto;
+  margin-right: auto;
   padding: 0 var(--rm-space-large);
   width: 100%;
 }
 
+.container--xs {
+  max-width: $breakpoint-x-small;
+}
+
 .container--sm {
   max-width: $breakpoint-small;
 }
 
+.container--sm-padding {
+  padding: 0 var(--rm-space-small);
+}
+
 .container--md-padding {
   padding: 0 var(--rm-space-medium);
 }
 
-.container--sm-padding {
-  padding: 0 var(--rm-space-small);
+// Section Properties
+
+.section-sm {
```

Comment:
I'm not a fan of having `CSS` classes that are mirrored to standard `HTML` tags. In this case  `section`, and the related `section-sm`, and `section-lg`.